src/controllers/accounts.js

```diff
@@ -14,6 +14,13 @@ exports.getCollection = function * find () {
   this.body = _.invoke(accounts, 'getDataExternal')
 }
 
+exports.getConnectors = function * () {
+  const accounts = yield Account.findAll({
+    where: { connector: { $ne: null } }
```

Comment:
It isn't possible for it to be `undefined`. It could be `''` if it was explicitly set to that by the admin, but I think that's on them.